### ISSUE TITLE
Sitemap module

### ISSUE THREAD

**Issue by finnlewis** [localgovdrupal/localgov#49]
Include sitemap module
See https://github.com/localgovdrupal/localgov_services/issues/11

**Comment by finnlewis** [comment]
I've had issues getting the configuration for simple_sitemap module to install in the desired manner. 

See related pull requests:
https://github.com/localgovdrupal/localgov/pull/50 
https://github.com/localgovdrupal/localgov_services/pull/53

We would ideally like the config to inclde each content type in the sitemap to be optional, depending on the simple_sitemap modile being installed. 

This is how it is currently trying to work, for example on the localgov_guide content types: 
https://github.com/localgovdrupal/localgov_guides/blob/master/config/optional/simple_sitemap.bundle_settings.default.node.localgov_guides_overview.yml

Basically, this is not importing.

Interestingly it does import if the config is in the /config/install directory, just not if it is in the /config/optional folder. 

I've posted an issue on drupal.org: https://www.drupal.org/project/simple_sitemap/issues/3156080

**Comment by ekes** [comment]
If it works in config/install when simple_sitemap isn't installed? But what about install order?

My thoughts were:
 * Move to profile, but still have to handle optional modules enabled later.
Or maybe better:
 * `\Drupal::service('simple_sitemap.generator')->setBundleSettings('node', $content_type, ['index' => TRUE, 'priority' => 0.5])` in install and modules_installed hooks?
